content/en/user-guide/ci/circle-ci/index.md

```diff
@@ -1,45 +1,118 @@
-The [Orb's documentation](https://circleci.com/developer/orbs/orb/localstack/platform) features examples, as well as a description of the available commands.
+This guide is designed for users new to CircleCI and assumes basic knowledge of YAML and LocalStack tooling. To create a CircleCI job that uses LocalStack, follow these steps:
 
-When using the official CircleCI Orb, using LocalStack in your pipeline is as easy as adding the Orb to your pipeline and executing the startup command.\
-The following example CircleCI config (`.circleci/config.yml`) starts LocalStack, creates a new S3 bucket, and prints a nice message in the end:
-```yaml
+- Under the "Projects" tab, find your project and click "Set Up Project"
+- You'll be prompted to add a configuration. You can add the configuration manually or choose a starter pipeline.
+- If you choose the starter CI pipeline, a sample `config.yml` file is created and committed to a `circleci-project-setup` branch in your repo.
+- You can add the following configuration to the `config.yml` file to start LocalStack and run your tests against it.
+
+{{< tabpane >}}
+{{< tab header="Community" lang="yml" >}}
 version: 2.1
+
 orbs:
-  localstack: localstack/platform@1.0
+  python: circleci/python@2.0.3
+
 jobs:
-  run-integration-tests:
-    executor: localstack/default
+  example-job:
+    machine:
+      image: ubuntu-2004:2022.04.1
+
     steps:
-      - localstack/startup
+      - checkout
+
       - run:
-          command: awslocal s3 mb s3://test-bucket
+          name: Start LocalStack
+          command: |
+            pip3 install localstack awscli-local[ver1]
+            docker pull localstack/localstack
+            localstack start -d                     
+
+            echo "Waiting for LocalStack startup..."  
+            localstack wait -t 30                     
+            echo "Startup complete"
+            
       - run:
-          command: echo "Execute your tests here :)"
+          name: Run AWS CLI commands against LocalStack
+          command: |
+            awslocal s3 mb s3://test-bucket
+            awslocal sqs create-queue --queue-name test-queue
+            awslocal sns create-topic --name test-topic
+
 workflows:
-  integration-test:
+  version: 2
+  build:
     jobs:
-      - run-integration-tests
-```
+      - example-job
+{{< /tab >}}
+{{< tab header="Pro" lang="yml" >}}
+version: 2.1
+
+orbs:
+  python: circleci/python@2.0.3
+
+jobs:
+  example-job:
+    machine:
+      image: ubuntu-2004:2022.04.1
+    environment:
+      LOCALSTACK_API_KEY: YOUR_API_KEY_HERE
+
+    steps:
+      - checkout
+
+      - run:
+          name: Start LocalStack
+          command: |
+            pip3 install localstack awscli-local[ver1]
+            docker pull localstack/localstack-pro
```

Comment:
nit: Should technically not be required - I'd personally keep the docs clean of any manual `docker pull` commands, so we can control this from the CLI. The main exception would be cases where we're pulling an image asynchronously in the background across build steps, for optimization reasons.

(Same for the command on line 39 above)